zhaquirks/tuya/ts0121_plug.py

```diff
@@ -62,3 +64,65 @@ class Plug(CustomDevice):
             },
         },
     }
+
+
+class TS0121B(CustomDevice):
+    """Tuya TS0121 plug with restore tuya power state support and ZGP endpoint."""
+
+    signature = {
+        MODEL: "TS0121",
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=81
+            # device_version=1
+            # input_clusters=[0, 4, 5, 6, 9, 1794, 2820]
```

Comment:
The device signature is wrong.
Maybe copy&paste?

Reply:
`input_clusters` comment is wrong for both devices too. Neither defines 0x0009 "Alarms" and the new one is obviously completely different...

Reply:
fixed